sale_outplacement/models/outplacement.py

```diff
@@ -81,38 +85,36 @@ def _get_partner_id(self, data):
 
     @api.multi
     def _get_management_team_id(self, data):
-        ResPartner = self.env['res.partner']
-        partner = ResPartner.search([
-            ('phone', '=', data['telefonnummer_handlaggargrupp'])
-        ])
-        if not partner:
-            ResPartner.create({
+        employee = self.env['hr.employee'].search(
+            [('work_phone', '=', data['telefonnummer_handlaggargrupp'])], limit=1)
+
+        if not employee:
+            employee = self.env['hr.employee'].create({
```

Comment:
During creation of an employee you might want more fields than this and a connected user if they are intended to use the system. This looks like some kind of placeholder due to lack of data given that the name is set to the email, but maybe the placeholder nature of the employee should be further clarified, maybe set lastname to "placeholder".